### PR TITLE
[OBSDEF-5644] Added prop to show/hide datagrid filter

### PR DESCRIPTION
**Pull request by payald**
Description: 
Added prop in DatagridFilter to show or hide filter.
Added className in Toggle.

Screenshot:
image

### CODE REVIEW

File: src/datagrid/DataGridFilter.tsx [hunk 1]
```diff
@@ -93,7 +94,7 @@ export class DataGridFilter extends React.PureComponent<DataGridFilterProps, Dat
     };
 
     // Initial state for filter
-    state: DataGridFilterState = {
+    state = {
```
Comment (akate1) [inline src/datagrid/DataGridFilter.tsx:97]: can you please add type to state ?

File: src/datagrid/DataGridFilter.tsx [hunk 2]
```diff
@@ -250,24 +251,27 @@ export class DataGridFilter extends React.PureComponent<DataGridFilterProps, Dat
...
     render() {
         const {isOpen} = this.state;
+        const {showFilter} = this.props;
         const {isFiltered} = this;
         const FilterBtnClasses = classNames([
             ClassNames.DATAGRID_FILTER_BUTTON,
             isFiltered && ClassNames.DATAGRID_FILTERED,
         ]);
-
+        const isFilterVisible = showFilter === undefined || showFilter;
```
Comment (akate1) [inline src/datagrid/DataGridFilter.tsx:260]: instate of this. make showFilter as true by default

**Review by girishdudhe — APPROVED** [review]
LGTM

**Review by ghatea — APPROVED** [review]
LGTM

**Review by maheshkulkarni4 — APPROVED** [review]
LGTM

**Pull request merged**